hello-pepr-alias/capabilities/alias.e2e.test.ts

```diff
@@ -15,273 +15,108 @@ describe("alias.ts", () => {
     await moduleDown();
   }, mins(2));
 
-  describe("create - reconcile - finalize", () => {
+  describe("provides correct alias", () => {
     let logz: string[];
-
     beforeAll(async () => {
       const file = `${trc.root()}/capabilities/scenario.create.yaml`;
       await timed(`load: ${file}`, async () => {
-        let [ns, cmReconcile, cmWatch, cmValidate, cmMutate] =
-          await trc.load(file);
+        let [ns, cmReconcile, cmWatch, cmValidate, cmMutate] = await trc.load(file);
         await fullCreate([ns, cmReconcile, cmWatch, cmValidate, cmMutate]);
-
         await K8s(kind[cmReconcile.kind]).Delete(cmReconcile);
         await K8s(kind[cmWatch.kind]).Delete(cmWatch);
         await K8s(kind[cmValidate.kind]).Delete(cmValidate);
         await K8s(kind[cmMutate.kind]).Delete(cmMutate);
-
+        await untilLogged(
+          '"msg":"external api call (reconcile-create-alias): reconcile/finalize"',
+        );
```

Comment:
You're running _all four resource actions at once_ here (reconcile AND watch AND validate AND mutate) but you're only waiting until you see the results of the _very first resource action_ (i.e. the reconcile), so... how do you know that the remaining 3 (watch, validate, mutate) resource actions ran successfully, or at all?

Because the way this is currently, you're setting up a race between how quickly the test cluster chews through your resource actions vs. how quickly your test runner processes assertions.